ArjunaJTS/standalone/start.sh

```diff
@@ -53,9 +53,9 @@ function startRM {
 # start client
 function startClient {
   if [ $ORBTYPE = "jacorb" ]; then
-    mvn compile exec:java  -Dexec.cleanupDaemonThreads=false -Dexec.mainClass=org.jboss.narayana.jta.quickstarts.RemoteTMExample -DOrbPortabilityEnvironmentBean.orbImpleClassName=com.arjuna.orbportability.internal.orbspecific.jacorb.orb.implementations.jacorb_2_0 -DNAME_SERVER_HOST="$HOST_ADDRESS"
+    mvn compile exec:java  -Pscript -Dexec.cleanupDaemonThreads=false -Dexec.mainClass=org.jboss.narayana.jta.quickstarts.RemoteTMExample -DOrbPortabilityEnvironmentBean.orbImpleClassName=com.arjuna.orbportability.internal.orbspecific.jacorb.orb.implementations.jacorb_2_0 -DNAME_SERVER_HOST="$HOST_ADDRESS"
```

Comment:
Same here. This `if` should not be needed as we have only one possible choice (i.e. openjdk-orb)

Reply:
An aside that I think it may be possible we would work with `ibmorb` for IBM JDK 11 still (assuming IBM's JDK 11)  has one but I don't know - I don't think it's present in IBM JDK 17. That being said it doesn't look like https://github.com/jbosstm/quickstart/blob/master/ArjunaJTS/standalone/start.sh has reference to ibmorb anyway.